src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Apprenticeship.cs

```diff
@@ -245,14 +245,22 @@ public void ReplacePriceHistory(Party party, List<PriceHistory> currentPriceHist
             this.PriceHistory = updatedPriceHistory;
         }
 
-        public void UpdateCourse(Party party, string courseCode, string courseName, ProgrammeType programmeType, UserInfo userInfo)
+        public void UpdateCourse(Party party, string courseCode, string courseName, ProgrammeType programmeType, UserInfo userInfo, string standardUId)
         {
             StartTrackingSession(UserAction.UpdateCourse, party, Cohort.EmployerAccountId, Cohort.ProviderId, userInfo);
             ChangeTrackingSession.TrackUpdate(this);
 
             CourseCode = courseCode;
             CourseName = courseName;
             ProgrammeType = programmeType;
+            StandardUId = standardUId;
```

Comment:
You wouldn't know this, but I think we need to to clear the Options field as well. If the provider has assigned an option, prior to this step, it will be wrong and has to be reset to null

Reply:
Paul re these 'Options'.. I can see a List of strings is returned back from TrainingProgramme as an Options collection. In the case of my local test data, its an empty collection - but I assume this will be populated at some point. Are you suggesting that if this collection comes back populated, we null null this list? Or are you referring to setting TrainingCourseOption value and passing that collection through, we can do that...

Reply:
No, it's just a value in the commitments DB. The apprenticeship record contains a field called TrainingCourseOption. We simply need to blank this value to NULL in your ChangeCourse Code. Because we are changing the course, the option is probably no longer valid